jenkins/job-configs/bootstrap-maintenance.yaml

```diff
@@ -80,6 +80,10 @@
             </hudson.plugins.parameterizedtrigger.TriggerBuilder>
     triggers:
     - timed: '{frequency}'
+    wrappers:
+    - timeout:
+        timeout: 180
```

Comment:
We need different timeouts for the hourly job (180 is fine there) and the daily job (for this we should make it at least 30 hours)

Use something like `timeout: {all-timeout}` and then add it to the `- daily` `- hourly` keys below

Reply:
hm, I'm now realizing we already have a bootstrap-enforced timeout of 30m for the actual jobs. 

do we expect to queue for more than 3h for any of the maintenance jobs?

Reply:
Is time in the queue counted towards jenkins timeout? (I'd expect not?)

Reply:
Start time is when the job starts, not when it is queued.